### PR TITLE
FAIE-1673: support inference for the latest anomalib

### PR DESCRIPTION
**Pull request by nickjyj**
Tested tensorRT and pytorch weights of three models on x86: padim, patchcore and efficientad. 

Haven't test on ARM because we don't have any orin devices with python 3.10.

### CODE REVIEW

**Comment by nickjyj** [comment]
Also, updated the pipeline_utils/resize_image function to return the same type as the input image

File: anomaly_detectors/anomalib_lmi/anomaly_model2.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+
+logging.basicConfig()
+
+
+MINIMUM_QUANT=1e-12
+Binding = namedtuple('Binding', ('name', 'dtype', 'shape', 'data', 'ptr'))
+
+
+class AnomalyModel2(Anomalib_Base):
```
Comment (RugvedMavidipalli) [inline anomaly_detectors/anomalib_lmi/anomaly_model2.py:21]: Probably better to name it AnomalyModelv2

**Review by RugvedMavidipalli — APPROVED** [review]
Only one comment and looks good.

**Review by caaden — APPROVED** [review]

**Review by trevorditrani — APPROVED** [review]

**Pull request merged**